packages/cli/src/commands/init/init.js

```diff
@@ -74,14 +79,12 @@ async function createFromExternalTemplate(
     const {postInitScript} = templateConfig;
     if (postInitScript) {
       // Leaving trailing space because there may be stdout from the script
-      loader.start('Executing post init script ');
```

Comment:
please revert

Reply:
Completely missed the comment.

![](https://media1.tenor.com/images/4b50f11afd13042d2e74f3cef023ba6c/tenor.gif?itemid=5471587)

Thanks for seeing this.